packages/autoloader/README.md

```diff
@@ -0,0 +1,56 @@
+A custom autoloader for composer
```

Comment:
https://gist.github.com/jeherve/8123a50df5cb5d00b67af6ebaec95374

Reply:
❤️ this ^

Reply:
thanks. fixed